src/collections/_documentation/workflow/integrations/jira-server.md

```diff
@@ -0,0 +1,88 @@
+---
+title: JIRA Server
+sidebar_order: 9
+---
+
+Connect errors from Sentry with your Jira Server issues.
+
+## Installing Jira Server with Sentry
+
+{% capture __alert_content -%}
+Sentry owner or manager permissions, and Jira administrator permissions are required to install this integration.
+{%- endcapture -%}
+{%- include components/alert.html
+  title="Note"
+  content=__alert_content
+  level="warning"
+%}
+
+### I. Generate an RSA public/private key pair
+Complete the instructions in the section titled **Generate an RSA public/private key pair** in [these Jira docs](https://developer.atlassian.com/server/jira/platform/oauth/). 
+You'll need your public and private keys for later in the process.
+
+### II. Create a new application link in Jira
+1. In Jira, click the **gear icon** > **Applications** > **Application Links**.
+1. Enter the following as the application URL: 
+`https://sentry.io/extensions/jira_server/setup/`
+1. Click **Create New Link**. If you see a warning that “No response was received from the URL you entered”, ignore and click **Continue**.
+1. In the resulting dialog, fill out the form as follows:
+
+    | Application Name                 | Sentry        |
+    | Application Type                   | Generic Application |
+    | Service Provider Name | Sentry |
+    | Consumer Key                       | (your choice, but keep this handy for the next step) |
+    | Shared Secret                     | sentry |
+    | Request Token URL                  | https://sentry.io |
+    | Access Token URL | https://sentry.io    |
+    | Authorize URL           | https://sentry.io    |
+    | Create Incoming Link               | Yes    |
+1. Click **Continue**.
+1. On the next screen of the dialog, enter into the form the consumer key from the previous step, `Sentry` as the consumer name, and the public key created in Section I.
+1. Click **Continue**. You should now see an application link called Sentry.
+
+### III. Connect your Jira Server application with Sentry
+1. In Sentry, navigate to **Organization Settings** > **Integrations**.
+2. Next to Jira, click **Install**.
+3. In the resulting modal, click **Add Installation**.
+4. In the resulting window, enter the base URL for your Jira Server instance, your consumer key, and your private key. Click **Submit**.
+5. In Sentry, you’ll see a new Jira Server instance appear on the Integrations page.
+
+Jira should now be authorized for all projects under your Sentry organization.
+
+## Issue Management
+
+Issue tracking allows you to create Jira issues from within Sentry, and link Sentry issues to existing Jira Issues.
+
+{% capture __alert_content -%}
+Issue management is available for organizations on the Team, Business, and Enterprise plans.
```

Comment:
I thought all plans had issue create/link and higher plans were needed for sync.